Documentation/Troubleshooting/common-issues.md

```diff
@@ -2,38 +2,252 @@
+* Is the disk empty? No leftover partitions on it? Make sure it is either "empty", e.g., nulled by `shred`, `dd` or similar,
+    * To make sure the disk is blank as the Rook docs and I recommend the following commands followed by a reboot of the server:
+        ```
+        DISK="/dev/sdXYZ"
+        sgdisk --zap-all "$DISK"
+        dd if=/dev/zero of="$DISK" bs=1M count=100 oflag=direct,dsync
+        blkdiscard "$DISK"
+        ```
+        Source: [https://rook.io/docs/rook/v1.8/ceph-teardown.html#delete-the-data-on-hosts](https://rook.io/docs/rook/v1.8/ceph-teardown.html#delete-the-data-on-hosts)
+* Was the disk previously used as a Ceph OSD?
+    * Make sure to follow the teardown steps, but make sure to only remove the LVM stuff from that one disk and not from all, see [https://rook.io/docs/rook/v1.8/ceph-teardown.html#delete-the-data-on-hosts](https://rook.io/docs/rook/v1.8/ceph-teardown.html#delete-the-data-on-hosts).
+
+## A Pod can't mount its PersistentVolume after an "unclean" / "undrained" Node shutdown
+
+1. Check the events of the Pod using `kubectl describe pod POD_NAME`.
+2. Check the Node's `dmesg` logs.
+3. Check the kubelet logs for errors related to CSI connectivity and / or make sure the node can reach every other Kubernetes cluster node (at least the Rook Ceph cluster nodes (Ceph Mons, OSDs, MGRs, etc.)).
+4. Checkout the [CSI Common Issues - Rook Docs](https://rook.io/docs/rook/v1.8/ceph-csi-troubleshooting.html).
+
+## Ceph CSI: Provisioning, Mounting, Deletion or something doesn't work
+
+Make sure you have checked out the [CSI Common Issues - Rook Docs](https://rook.io/docs/rook/v1.8/ceph-csi-troubleshooting.html).
+
+If you have some weird kernel and / or kubelet configuration, make sure Ceph CSI's config options in the Rook Ceph Operator config is correctly setup (e.g., `LIB_MODULES_DIR_PATH`, `ROOK_CSI_KUBELET_DIR_PATH`, `AGENT_MOUNTS`).
+
+## Can't run any Ceph Commands in the Toolbox / Ceph Commands timeout
+
+* Are your `rook-ceph-mon-*` Pods all in `Running` state?
+* Does a basic `ceph -s` work?
+* Is your `rook-ceph-mgr-*` Pod(s) running as well?
+* Check the `rook-ceph-mon-*` and `rook-ceph-mgr-*` logs for errors
+* Try deleting the toolbox Pod, "maybe it is just a fluke in your Kubernetes cluster network / CNI.
+    * Also make sure you are using the latest Rook Ceph Toolbox YAML for the Rook Ceph version you are running on, see [Rook Ceph Toolbox Pod not Creating / Stuck section](#rook-ceph-toolbox-pod-not-creating--stuck).
+* In case all these seem to indicate a loss of quorum, e.g., the `rook-ceph-mon-*` talk about `probing` for other mons only, you might need to follow the disaster recovery guide for your Rook Ceph version here: [Rook v1.8 Docs - Ceph Disaster Recovery - Restoring Mon Quorum](https://rook.io/docs/rook/v1.8/ceph-disaster-recovery.html#restoring-mon-quorum).
```

Comment:
Why is this in an older version of rook?

Reply:
Fixed to be pointing directly at the docs files now, hopefully in all places now